lib/archethic_web/views/faucet_view.ex

```diff
@@ -0,0 +1,3 @@
+defmodule ArchEthicWeb.FaucetView do
```

Comment:
I don't think you need a specific view for this, you can leverage the explorer view.

`Phoenix.View.render(view, template, assigns)`